code/controllers/subsystem/air.dm

```diff
@@ -391,9 +393,14 @@ SUBSYSTEM_DEF(air)
 			//EG.dismantle()
 			CHECK_TICK*/
 
-		var/msg = "HEY! LISTEN! [DisplayTimeText(world.timeofday - timer)] were wasted processing [starting_ats] turf(s) (connected to [ending_ats] other turfs) with atmos differences at round start."
-		to_chat(world, span_boldannounce("[msg]"))
-		warning(msg)
+		//Yogs start -- prettier atmos notices
+		var/msg = "HEY! LISTEN! [(world.timeofday - timer)/10] seconds were wasted processing [starting_ats] turf(s) (connected to [ending_ats] other turfs) with atmos differences at round start."
```

Comment:
its meant to be annoying please just fix the atmospherics

Reply:
If anything, the solution would be for the thing that's complaining to have some option to write a new map file post-equalization that can just be saved.

Provided anyone really does care about the, <100 ms that this is adding to load times on Box. Not a big performance target + out of scope of this PR + looks bad

Reply:
I like that idea